server_admin/topics/authentication/webauthn.adoc

```diff
@@ -0,0 +1,197 @@
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#dictdef-publickeycredentialparameters[WebAuthn Specification].
+
+|Relying Party ID
+|This is ID as WebAuthn Relying Party and determines the scope of Public Key Credentials. It must be origin's effective domain. This is optional configuration item and applied to the operation of registering WebAuthn authenticator. If no entry is entered, the host part of the base URL of {project_name}'s server is adapted.
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#rp-id[WebAuthn Specification].
+
+|Attestation Conveyance Preference
+|It tells the WebAuthn API implementation on the browser (https://www.w3.org/TR/webauthn/#webauthn-client[WebAuthn Client]) the preference of how to generate an Attestation Statement. However, {project_name} does not verify Attestation Statement so that only "none" option can be selected. This is optional configuration item and applied to the operation of registering WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "none".
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#attestation-convey[WebAuthn Specification].
+
+|Authenticator Attachment
+|It tells the WebAuthn Client an acceptable attachment pattern of a WebAuthn authenticator. This is optional configuration item and applied to the operation of registering WebAuthn authenticator. If no option is selected, the WebAuthn Client does not consider the attachment pattern.
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#enumdef-authenticatorattachment[WebAuthn Specification].
+
+|Require Resident Key
+|It tells the WebAuthn authenticator to generate the Public Key Credential as https://www.w3.org/TR/webauthn/#client-side-resident-public-key-credential-source[Client-side-resident Public Key Credential Source]. This is optional configuration item and applied to the operation of registering WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "No".
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#dom-authenticatorselectioncriteria-requireresidentkey[WebAuthn Specification].
+
+|User Verification Requirement
+|It tells the WebAuthn authenticator to confirm actually verifying a user. This is optional configuration item and applied to both the operation of registering WebAuthn authenticator and authenticating the user by WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "preferred".
```

Comment:
Try: "This is an optional configuration item that is applied to the operation of registering WebAuthn authenticator and authenticating the user by WebAuthn authenticator."

Reply:
Thank you. I'll fix this sentence as you suggested.